bootstrap_cfn/config.py

```diff
@@ -92,6 +76,10 @@ def process(self):
         template['Outputs'] = {}
         for t in output_templates:
             template['Outputs'].update(json.loads(pkgutil.get_data('bootstrap_cfn', t)))
+        if 'includes' in self.data:
+            for inc_path in self.data['includes']:
+                inc = json.load(open(inc_path))
+                utils.dict_merge(template, inc)
```

Comment:
You aren't doing anything with the return value from `dict_merge` - which means that it's merging in place into template - if that's the case can you remove the return statement from that function and add a simple doc comment.